### ISSUE TITLE
Support for statuses (stories)

### ISSUE THREAD

**Issue by mugavri** [pedroslopez/whatsapp-web.js#2371]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Describe the solution you'd like.

Adding support for statuses (stories)
- Receiving my list of statuses.
- Downloading the media.
- Getting the list of users who viewed the status.
- Adding the information in the message when commenting on a status.
- Adding an event when viewing a new one.

**Comment by mugavri** [comment]
> Not possible while using puppeteer, because it uses from whatsapp web, except if that features has available in whatsapp web

These features exist in whatsapp web

**Comment by RC047** [comment]
> > Not possible while using puppeteer, because it uses from whatsapp web, except if that features has available in whatsapp web
> 
> These features exist in whatsapp web

Yes, I mean the feature of whatsapp stories is rudimentary in whatsapp web, like adding and deleting stories

**Comment by alechkos** [comment]
Duplicate of #426

**Comment by lord-zeus** [comment]
> > Not possible while using puppeteer, because it uses from whatsapp web, except if that features has available in whatsapp web
> 
> These features exist in whatsapp web

Just the View Status will be ok
